pydictable/core.py

```diff
@@ -100,12 +100,16 @@ def __get_field_by_type_hint(cls, type_hint):
     @classmethod
     def get_fields(cls) -> Dict[str, Field]:
         fields = {}
-        for attr in inspect.getmembers(cls):
-            if isinstance(attr[1], Field):
-                fields[attr[0]] = attr[1]
+        dictable_fields = {k: v for k, v in dict(vars(cls)).items() if isinstance(v, Field)}
         for name, th in get_type_hints(cls).items():
```

Comment:
Can we not change the existing implementation? Something like this
```python
fields = {}
...
ordered_fields = {}
partial_ordered_fields = {... dict(vars())}
for k, field in partial_ordered_fields:
    if k in fields:
        ordered_fields[k] = field
for k, field in fields:
    if k not in ordered_fields:
        ordered_fields[k] = field
assert set(fields.keys()) == set(ordered_fields.keys())
return ordered_fields
```

Reply:
Updated. Have also removed using `inspect.getmembers()` since it's not needed